diesel/src/pg/expression/extensions/only_dsl.rs

```diff
@@ -0,0 +1,41 @@
+use crate::query_builder::Only;
+use crate::Table;
+
+/// The `only` method
+///
+/// This is only implemented for the Postgres backend.
+/// The `ONLY` clause is used to select only from one table and not any inherited ones.
+///
+/// Calling this function on a table (`mytable.only()`) will result in the SQL `ONLY mytable`.
+/// `mytable.only()` can be used just like any table in diesel since it implements
+/// [Table](crate::Table).
+///
+/// Example:
+///
+/// ```ignore
```

Comment:
I would prefer not to introduce new `ignore` doc tests. We should at least check that any code in our documentation compiles.

(Same applies for the test below)

Reply:
Is there a module from which I can import the `users` and `posts` tables? I think it's only in `diesel_tests::schema` - what do I do, define them anew?

Reply:
https://github.com/diesel-rs/diesel/blob/c237a74b63900b734c659a201fc1d6b87108033b/diesel/src/doctest_setup.rs 

which is included via `include!()` in other doc tests. See here for example https://github.com/diesel-rs/diesel/blob/c237a74b63900b734c659a201fc1d6b87108033b/diesel/src/insertable.rs#L42
(You likely need to adjust the path, as it's relative to the current source file.)